index.js

```diff
@@ -164,7 +164,7 @@ function runChromeDevTools(context) {
 
   var testHeader = 'Chrome A11Y - ';
 
-  return browser.executeScript_(data, 'a11y developer tool rules').then(function(results) {
+  return browser.executeScript(data, 'a11y developer tool rules').then(function(results) {
```

Comment:
Thanks for the fix! Could you make this `browser.executeScriptWithDescription`? That's what `browser.executeScript_` got renamed to.